tensorflow_text/python/ops/wordpiece_tokenizer.py

```diff
@@ -51,6 +52,9 @@ def __init__(self,
       suffix_indicator: (optional) The characters prepended to a wordpiece to
         indicate that it is a suffix to another subword. Default is '##'.
       max_bytes_per_word: (optional) Max size of input token. Default is 100.
+      max_bytes_per_token: (optional) Max size of subwords, excluding suffix
```

Comment:
should this be max_chars_per_token instead?

Reply:
the way I have implemented it, 'bytes' is correct. Would you like me to instead implement it as max chars? It would make the logic in `LongestMatchStartingAt` a little less clean

Reply:
max_chars_per_token would helpful to support i18n languages and emojis (where the number of bytes can vary). for example, we could do unicode character splits in chinese:

wp = WordpieceTokenizer(..., suffix_indicator="", max_chars_per_token=1)
wp.tokenize(["蜘蛛人"]) => [["蜘", "蛛", "人"]]

Reply:
Note to get that behaviour we would need my other PR: https://github.com/tensorflow/text/pull/73 . setting just `max_chars_per_token=1` in the context of this PR would just do `wp.tokenize(["蜘蛛人"]) => [["UNK"]]`
Still I'll change it to `max_chars_per_token`, just want to clarify

Reply:
Latest commit changes to `max_chars_per_token`